internal/metrics/prometheus.go

```diff
@@ -52,9 +63,12 @@ func NewCtrlFuncMetrics() *CtrlFuncMetrics {
 				ProtocolKey + "` describes the configuration protocol (" + ProtocolDBLess + " or " +
 				ProtocolDeck + ") in use. `" +
 				SuccessKey + "` describes whether there were unrecoverable errors (`" +
-				SuccessFalse + "`) or not (`" + SuccessTrue + "`).",
+				SuccessFalse + "`) or not (`" + SuccessTrue + "`). `" +
+				FailureReasonKey + "` is populated in case of `" + SuccessKey +
+				"=\"" + SuccessFalse + "\"` " + "and describes the reason of failure (one of `" +
+				FailureReasonConflict + "`, `" + FailureReasonOther + "`).",
 		},
```

Comment:
Can we maybe rewrite this whole string formatting thingy into a nicely formatted `fmt.Sprintf()` invocation?